packages/mail/utils/sendResultNormalizer.js

```diff
@@ -0,0 +1,35 @@
+const sleep = require('await-sleep')
+
+module.exports = (shouldSend, sendFunc) =>
+  async () => {
+    let results, error
+    if (shouldSend) {
+      try {
+        results = await sendFunc()
+      } catch (e) {
+        error = e
+        console.warn('send failed', error)
+        return {
+          result: (results && results[0]) || null,
+          status: 'FAILED',
+          error
+        }
+      }
+    } else {
+      await sleep(1000)
+      results = [{ status: 'sent-simulated' }]
+    }
+    const result = results && results[0]
+    const wasSent = (
+      result && !error && (
+        (result.status === 'sent' || !result.status === 'sent-simulated') &&
```

Comment:
accidental bang? `!result.status === 'sent-simulated'` will never be true because `true !== 'sent-simulated' && `false !== 'sent-simulated'`.